src/index.ts

```diff
@@ -109,6 +109,12 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "Path to a current working directory",
             requiresArg: true,
         })
+        .option("local-variables-file", {
```

Comment:
naming is hard, let me know if you want to name this something else

Reply:
`gitlab-ci-local --variables-file=/home/mjn/something-custom/variables.yml`

No need to emphasize that its local, you are running `gitlab-ci-LOCAL` already.